### PR TITLE
Expand keywords

### PR DESCRIPTION
**Pull request by reubenjh**
Adds/extends
- mortal
- undead
- beast
- spirit
- dwarf
- gnome
- angel
- voidbound
- flood
- armor
- weapon
- dragon
- monster
- burrowing
- submerge
- charge
- demon
- monument

Removes
- armor_x
- fire_spellcaster

### CODE REVIEW

File: data/en/cards/alpha/cards.yaml [hunk 1]
```diff
@@ -1564,6 +1630,8 @@
   card_type: magic
   release: alpha
   artist: jussi_pylkäs
+  keywords:
+  - submerge
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1634]: This is questionable. I've added the submerge keyword to all cards that have an effect that surface submerged minions. Whatcha reckon?

File: data/en/cards/alpha/cards.yaml [hunk 2]
```diff
@@ -3601,7 +3762,6 @@
   artist: ian_miller
   keywords:
   - spellcaster
-  - fire_spellcaster
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3604]: What do you think about removing this sub-keyword? It only has two instances, is already covered by spellcaster, and there's also cards with air spellcaster that isn't set up as a keyword. I think if we keep this one we have to make air_spellcaster too

File: data/en/cards/alpha/cards.yaml (hunk 1 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:1648]: Let's go for it for now. I think we probably need to circle back to model keyword-adjacentness as more of a deckbuilding concept that we can tag things with.

File: data/en/cards/alpha/cards.yaml (hunk 2 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:3604]: I like the call 👍

**Comment by tonywok** [comment]
Thanks dude. I'll fix up the test and merge.

**Pull request merged**